pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/data/repository/UserRepository.java

```diff
@@ -94,6 +90,7 @@ public User findOrCreate(User inputUser) {
             user = createUser(inputUser);
         } catch (NonUniqueResultException e) {
             logger.error("findOrCreate() " + e.getClass().getSimpleName() + ": " + e.getMessage());
+            user = createUser(inputUser);
```

Comment:
@Luke-Sikina and @srpiatt This was a mistake. I should have removed this one and only kept the one above.
```suggestion
```